### ISSUE TITLE
Operate on more than one word

### ISSUE THREAD

**Issue by GoogleCodeExporter** [minman/browse-word-at-caret#2]
```
The plugin defaults to searching single words, I'd love the ability to search 
for more, eg I select a class in HTML:

<div class="some-class"></div>

and select `some-class` but upon pressing alt+ctrl+up it only selects either 
`some` or `class`. 

Would it be possible to for example call the default select word at caret 
before searching, so the selection would expand to closest whole expression. 
Basically I'd love to use this as the default `highlight selected text` 
notepad++ functionality plus the ability to traverse through occurrences, or 
should I use something else entirely?
```

Original issue reported on code.google.com by `Raveren` on 17 May 2012 at 12:51
